specs/0108-community-health-dashboard.md

```diff
@@ -0,0 +1,49 @@
+
+# Motivation
+
+Team managers as well as project managers should have the ability to understand how localization communities are performing. Team dashboards already provide some of the answers, including the overall status of completion and translation statistics, but the information is only available in the form of a current snapshot.
+
+There's no historic data to provide insights into how the community health is developing. Missing is the information on how completion and volume of unreviewed strings are changing over time. It's not clear what share of translations was copied from the Machinery. We also don't know the ratio of translations approved without a peer review.
+
+# Out of scope
+
+At this stage we’re only interested in presenting data we already collect in Pontoon. We're primarily focusing on exposing data to team managers; project managers will be able to obtain data across all teams if needed, e.g. by using scripts and the API.
+
+# Feature explanation
+
+A new "Insights" tab is made available on the Team page, positioned next to the "Contributors" tab. It consists of several sections, presenting data for the period of the last 12 months. Each section is described below. Fore more details about the design, see the [Mockup](#mockup) section.
+
+## Active users
+
+Pie charts showing ratios of active vs. all managers, reviewers and contributors. See the existing scripts ([1](https://github.com/flodolo/scripts/blob/954fa85/pontoon/active_contributors.py), [2](https://github.com/flodolo/scripts/blob/954fa85/pontoon/list_reviewers_with_contribution_stats.py)) for criteria of defining active users and for details on how to retrieve data.
+
+## Unreviewed suggestion lifespan
+
+A line chart showing time before suggestion gets approved or rejected. See the [existing script](https://github.com/flodolo/scripts/blob/954fa85/pontoon/unreviewed_suggestions_lifespan.py) for details on how to retrieve data. A tooltip showing the exact data at a given time appears when hovering over a chart.
+
+## Translation activity
+
+A combination of a line chart showing completion over time and a column chart showing approved translation submissions and source string additions over time in two separate columns. The translation submission column is a stack of two columns - human translations (`Translation.machinery_sources` not set) and Machinery translations (`Translation.machinery_sources` set). A tooltip showing the exact data at a given time appears when hovering over a chart.
+
+## Review activity
+
+A combination of a line chart showing the number of unreviewed suggestions over time and a column chart showing review actions and suggestion submissions over time in two separate columns. The review actions column is a stack of three columns - peer-approvals, self-approvals and rejections. A tooltip showing the exact data at a given time appears when hovering over a chart.
```

Comment:
What does the added column represent and what is its purpose?

Reply:
"Suggestion submissions". Maybe New or Submitted would be more self-explanatory than Added?

The chart shows how the amount on Unreviewed suggestions changes over time. Without showing newly added suggestions, it could be confusing to see Unreviewed suggestions grow for example, despite the evident review actions.

Reply:
What other ways would cause unreviewed suggestions to grow? Is the uptick self-explanatory?

Reply:
I'd leave that for locale managers to tell, because I'm biased. 😄

Note that each data series can be enabled or disabled by clicking on the corresponding item in the legend. They can also be disabled by default.